main.rb

```diff
@@ -0,0 +1,576 @@
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
```

Comment:
__[development][rubocop]__
* Method `Object#baz` is defined at both /sideci/sandbox2/main.rb:11 and /sideci/sandbox2/main.rb:56.